server/src/database.js

```diff
@@ -0,0 +1,139 @@
+const Sequelize = require('sequelize');
+
+const createStore = () => {
+  const sequelize = new Sequelize({
+    dialect: 'sqlite',
+    storage: './database.sqlite',
+  });
+
+  sequelize.authenticate().then(() => {
+    console.log('Connection to the database has been established successfully');
+  }).catch(err => {
+    console.error(`Unable to connect to the database: ${err}`);
+  });
+
+  const Model = Sequelize.Model;
+
+  class User extends Model {
+  }
```

Comment:
Add a TODO if more stuff will be added in the class. If not, just do {} instead of the newline.

Reply:
Same below

Reply:
It is just a class definition. Fixed it. Thank you!